spec/coverage_helper.rb

```diff
@@ -0,0 +1,14 @@
+if ENV['CI']
+  require 'active_support/core_ext/kernel/reporting'
+
+  # Require all ruby files for accuracte test coverage reports
+  %w(app gems lib spa_ui).each do |path|
```

Comment:
@matthewd For you second comment...are you suggesting we turn off coveralls for everything else but vmdb?